DoSomethingArchive/legacy-website
# Feature export: Add Sponsor and partner field

#### What's this PR do?

Adds the `Sponsor and Partners` field collection to field pages. This only adds the field, does not deal with displaying it yet. 
#### Where should the reviewer start?

`dosomething_fact_page.features.field_instance.inc` but this is just a feature export.
#### How should this be manually tested?

Go to create or edit a fact page and you should see the "Sponsor and Partners" field at the bottom of the edit page.
#### Any background context you want to provide?

For the Yahoo partnership.
#### What are the relevant tickets?

Addresses #5038
